v1/pipeline.json

```diff
@@ -28951,6 +28953,15 @@
                   "minLength" : 1
                 } ]
               },
+              "environmentVariables" : {
```

Comment:
in V1, we should call these `env_vars`

You can refer to io.harness.steps.shellscript.v1.ShellScriptStepInfoV1